internal/archive/fsops.go

```diff
@@ -426,7 +509,7 @@ func getRelativePath(fullPath, basePath string) string {
 	if basePath == "" {
 		return fullPath
 	}
-	return strings.TrimPrefix(fullPath, basePath)
+	return strings.TrimPrefix(strings.TrimPrefix(fullPath, basePath), string(filepath.Separator))
```

Comment:
Shouldn't the trim of the first separator be done only if `basePath` was a prefix of `fullPath`? To prevent changing the path in case it isn't actually relative to `basePath`

Reply:
added check